tests/js/enactorsTests.js

```diff
@@ -0,0 +1,46 @@
+/*!
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+(function ($, fluid) {
+    "use strict";
+
+    fluid.registerNamespace("gpii.tests");
+
+    /**************************
+     * Language Enactor Tests *
+     **************************/
+
+    fluid.defaults("gpii.tests.enactor.lang", {
+        gradeNames: ["gpii.firstDiscovery.enactor.lang", "autoInit"],
+        model: {
+            lang: "en"
+        },
+        invokers: {
+            reloadPage: {
+                funcName: "jqUnit.assert",
+                args: ["The reloadPage invoker should have fired"],
+                // As the configuration for invokers are merged rather than replace. The lines below is to override
+                // the component configuration in the defaults block for gpii.firstDiscovery.enactor.lang
```

Comment:
Some grammatical changes.

```javascript
                // As the configuration for invokers are merged rather than replaced, the lines below are to override
                // the component configuration in the defaults block for gpii.firstDiscovery.enactor.lang
```